OWNERS_ALIASES

```diff
@@ -2,18 +2,17 @@
 
 aliases:
   sig-cluster-lifecycle-leads:
+    - fabriziopandini
     - justinsb
-    - luxas
+    - neolit123
     - timothysc
   cluster-api-admins:
-    - davidewatson
-    - detiber
     - justinsb
+    - vincepri
   cluster-api-maintainers:
     - CecileRobertMichon
-    - detiber
```

Comment:
@detiber I copied pasted from CAPI, but if you want to stick around in CAPG, I'm happy to amend